src/resources/i18n/translations/en.json

```
@@ -618,10 +615,10 @@
 	"You must verify your email address to login Press Send Email then check your inbox of a verification email": "Verify your e-mail address to log in. Press \"Send e-mail\" and click it in your inbox.",
 	"You need to login to make clips": "Log in to make clips",
 	"You should receive an email shortly The email may go to your Spam folder": "You should receive an e-mail shortly. It may be in your spam.",
-	"Your free trial expired": "Your gratis Podverse premium ended ",
-	"Your free trial expires": "Your gratis Podverse period period will end ",
-	"Your membership expired": "Your Podverse premium ended ",
-	"Your premium membership expires": "Your Podverse premium will end ",
+	"Your free trial expired": "Your free trial Podverse Premium ended ",
+	"Your free trial expires": "Your free trial Podverse period period will end ",
```

Interesting sentence.
```suggestion
	"Your free trial expires": "Your Podverse Premium will end ",
```

---

Can you change it to `"Your Podverse Premium membership will end"`?

In theory we could sell multiple tiers of memberships, although I doubt we would ever go that route (unless it is a completely different service, like a membership geared towards podcasters rather than podcast listeners).

---

I don't see "membership" adding anything there.
Moreover, I think this only applies for gratis Podverse Premium users, who don't have a membership.
People don't want to be in, or purchase memberships.

---

> People don't want to be in, or purchase memberships.

Can you substantiate this with data or anything to try to change my mind? People buy gym memberships. I like having memberships that give me access to services.

> Moreover, I think this only applies for gratis Podverse Premium users

I don't know what you mean by "gratis Podverse Premium users". There are "gratis users" and there are "premium users". The gratis users do not sign in to the Podverse server. The premium users do.

---

I mean those in the three month gratis period.
Gym memberships are understood, for a known service. They also grant status.
Online memberships are often book-clubs you can't get out of.
People like buying things outright, so here it is the opposite, for something that isn't known, with known bad actors calling it a membership.
Somewhere it should say that it doesn't auto-renew (but that should be opt-in).

---

Fair points. I need time to think about it. For now I'm leaving most of the "Podverse Premium" changes in, but keeping a few instances of "Podverse Premium membership".